applications/asset_tracker_v2/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Overlay to use nRF7002 EK on top of nrf9160 DK for Wi-Fi scanning
+
+# Disable modem traces as UART1 is disabled
+CONFIG_NRF_MODEM_LIB_TRACE=n
+
+# Does not work with buttons and LEDs
```

Comment:
Potential future improvement is to use the GPIO expander on the nRF9160 DK for LEDs and buttons when the EK is used